### DISCUSSION TITLE
C#의  yield break

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#947] [5. Q&A]
C#의 코루틴에서 yield break 이 typescript 에서 어떻게 사용하나요? 
yield break 을 사용하면 에러가 나서...
코루틴을 특정 구간에서 중단하려면 어떻게해야 하나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, yield break가 제대로 동작하지 않던 문제가 다음 패치에 수정될 예정입니다.
업데이트 완료시 깃헙 커뮤니티 또는 공식 가이드의 Package Release Note에 공지로 올라올 예정입니다.
감사합니다.

**Reply by Zepeto-TechEvangelist** [reply]
우선은 yield null로 대체해서 사용 가능합니다.
